app/javascript/availability/components/course_reserve_due_date.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import availability from '../index.js';
+
+const CourseReserveDueDate = (props) => {
+    const holding = props.holding;
+
+    if (availability.isOnCourseReserve(holding)) {
+        const dueDate = new Date(holding.dueDate);
+
+        const day = dueDate.toLocaleDateString('en-us', {
+            year: 'numeric',
+            month: 'numeric',
+            day: 'numeric',
+            timeZone: 'America/New_York'
```

Comment:
Do we know if `holding.dueDate` is in GMT?

Reply:
I believe it is, but I am not 100% certain. I had to add the `timeZone` param in a previous commit because the CI server was running in a different time zone than the catalog server, which was causing tests to fail on CI.